lib/Elastica/IndexTemplate.php

```diff
@@ -0,0 +1,118 @@
+<?php
+namespace Elastica;
+
+use Elastica\Exception\InvalidException;
+
+/**
+ * Elastica index template object.
+ *
```

Comment:
I suggest to add here the link to the es documentation: https://www.elastic.co/guide/en/elasticsearch/reference/1.6/indices-templates.html